lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "3.22.1",
+  "lerna": "2.11.0",
```

Comment:
Uffff

Reply:
It actually works with 3.22.1, but we first need to update CI machines